routes/announcement.py

```
@@ -0,0 +1,14 @@
+from flask import Blueprint, jsonify, request
+from services.announcement_service import get_announcements
+
+announcement_bp = Blueprint("announcement", __name__)
+
+@announcement_bp.route("/List", methods=["POST"])
+def list_announcements():
+    # 获取用户已关闭的公告ID列表，也可能没有请求体
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)